[Bug Report] [Build] [input] Build failed when try to import theme-chalk scss files

<!-- generated by https://issue.element-plus.org/ DO NOT REMOVE -->

Bug Type: **`Build`**

## Environment

- Vue Version: `3.2.19`
- Element Plus Version: `2.0.4`
- Browser / OS: `Chrome 96`
- Build Tool: `Vue CLI`

## Reproduction

### Related Component

- `el-input`

### Reproduction Link

[Github Repo](https://github.com/Malcolm-Zhou/element-plus-bug-demo)

### Steps to reproduce

1. import scss file to override variables.
```scss
@forward "element-plus/theme-chalk/src/common/var.scss" with (
	$border-radius: (
		"base": 20px,
	),
	$colors: (
      'primary': (
        'base': green,
      ),
  ),
);
@use "element-plus/theme-chalk/src/index.scss";
```
2. Just install packages and build the project.

### What is Expected?

Build sucessfully

### What is actually happening?

Build error

log:
```
> vue-cli-service build


 ERROR  Error: CSS minification error: Lexical error on line 1: Unrecognized text.

  Erroneous area:
1: 5px + map.get($input-font-size, "default") + map.get($input-padding-horizontal, "default")
^........^. File: css/app.92a24d42.css
Error: CSS minification error: Lexical error on line 1: Unrecognized text.

  Erroneous area:
1: 5px + map.get($input-font-size, "default") + map.get($input-padding-horizontal, "default")
^........^. File: css/app.92a24d42.css
    at D:\Downloads\element-plus-spa-forked\element-plus-bug-demo\node_modules\.pnpm\@intervolga+optimize-cssnano-plugin@1.0.6_webpack@4.46.0\node_modules\@intervolga\optimize-cssnano-plugin\index.js:106:21
    at async Promise.all (index 0)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! element-plus@0.1.0 build: `vue-cli-service build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the element-plus@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```
### Additional comments

I think the error exists in theme-chalk/src/input.scss because SCSS varaibles in calc() function shoud be inside "#{}"
After I add the #{}, the error won't happen again.

<!-- 如果你需要插入截图，请在这粘贴。 -->

--- a/packages/theme-chalk/src/input.scss
+++ b/packages/theme-chalk/src/input.scss
@@ -275,8 +275,10 @@
   }
 
   $input-padding-size-with-extra-icon: calc(
-    5px + map.get($input-font-size, 'default') +
-      map.get($input-padding-horizontal, 'default')
+    5px + #{map.get($input-font-size, 'default')} + #{map.get(
+        $input-padding-horizontal,
+        'default'
+      )}
   );
 
   @include m(suffix) {
@@ -313,8 +315,10 @@
       }
 
       $input-padding-size-with-extra-icon: calc(
-        5px + map.get($input-font-size, $size) +
-          map.get($input-padding-horizontal, $size)
+        5px + #{map.get($input-font-size, $size)} + #{map.get(
+            $input-padding-horizontal,
+            $size
+          )}
       );
 
       &.#{$namespace}-input--prefix {

